cmd/bblfsh-performance/parseandstore/parseandstore.go

```diff
@@ -8,12 +8,14 @@ import (
 	"github.com/bblfsh/performance/storage"
 	"github.com/bblfsh/performance/storage/file"
 	"github.com/bblfsh/performance/storage/influxdb"
-	"github.com/bblfsh/performance/storage/prom-pushgateway"
+	prom_pushgateway "github.com/bblfsh/performance/storage/prom-pushgateway"
 
 	"github.com/spf13/cobra"
 	"golang.org/x/tools/benchmark/parse"
 )
 
+const fileFilterPrefix = performance.FileFilterPrefix
```

Comment:
I know it's a stilly question, but why do we need it here, instead of directly use `performance.FileFilterPrefix`?

Reply:
Now using directly.
Done.